defaults/main.yml

```diff
@@ -76,6 +76,15 @@ neofs_ir__sidechain_fee: 500000000
 neofs_ir__mainchain_fee: 25000000
 neofs_ir__named_container_register_fee: 2500000000
 
+# Control API configuration
+neofs_ir__control_enabled: False
+neofs_ir__control_endpoint_port: '8090'
+neofs_ir__control_endpoint_host: "{{ neofs_ir__bind_address }}"
```

Comment:
Control should be bound to `localhost` by default

Reply:
Fixed